src/components/Form.spec.js

```diff
@@ -72,7 +72,7 @@ describe('Form', () => {
     userEvent.click(nextButton)
```

Comment:
Can I ask why you do not initialize an user event with `const user = userEvent.setup()`first? :) [See Testing Library docs for user event setup.](https://testing-library.com/docs/user-event/setup)

Reply:
I started this project long before the `userEvent` changed and when starting again, I didn't check the tests 😆